quality/transform_load_dlt.py

```diff
@@ -144,3 +144,21 @@ def quality_judiciary_silver():
         )
         .filter(F.col('amount').isNotNull())
     )
+
+# COMMAND ----------
+
+
+@dlt.table(name=f'quality_total_subnat_silver')
+def quality_total_subnat_silver():
+    bronze = dlt.read('quality_cci_bronze')
+    year_cols = list(col_name for col_name in bronze.columns if col_name.isnumeric())
+    return (bronze
+        .filter(F.lower(F.col('category')) == 'Subnational: Total spending'.lower())
```

Comment:
@nihaas21 any reason why you chose to use category instead of category_code here for the extraction?

Reply:
@weilu couple of reasons.
1. Following the format of previous codes where category was mentioned
2. My assumption is that the format of category_code is more prone to error than category
I did run a quick check on SQL editor to see if both of them extract same number of fields but happy to discuss more and make edits as necessary.

Reply:
Well, 1 is actually a reason to use category_code. Pull the code and see.